internal/flags/helpers.go

```diff
@@ -129,6 +136,14 @@ func doMigrateFlags(ctx *cli.Context) {
 }
 
 func init() {
+	if usecolor {
+		// Annotate all help categories with colors
+		cli.AppHelpTemplate = regexp.MustCompile("[A-Z ]+:").ReplaceAllString(cli.AppHelpTemplate, "\u001B[33m$0\u001B[0m")
+
+		// Annotate flag categories with colors (private template, so need to
+		// copy-paste the entire thing here...)
+		cli.AppHelpTemplate = strings.ReplaceAll(cli.AppHelpTemplate, "{{template \"visibleFlagCategoryTemplate\" .}}", "{{range .VisibleFlagCategories}}\n   {{if .Name}}\u001B[33m{{.Name}}\u001B[0m\n\n   {{end}}{{$flglen := len .Flags}}{{range $i, $e := .Flags}}{{if eq (subtract $flglen $i) 1}}{{$e}}\n{{else}}{{$e}}\n   {{end}}{{end}}{{end}}")
```

Comment:
Hmm, so I explictly tried to avoid overriding the help templates to make future updates of the library easier. They keep adding stuff upstream and it's good not to have it custom.

Reply:
I know, but they made teh damn subtemplate private.